packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -60,6 +60,23 @@ cd okta-app
 npm install @okta/okta-signin-widget
 ```
 
+To be able to require `@okta/okta-signin-widget` please install:
```

Comment:
Is this something newly introduced in version 6?

Reply:
See my comment about changing `import` to `require` for SIW.
Using `require` requires these steps